com.unity.netcode.gameobjects/Tests/Editor/XXHashTests.cs

```diff
@@ -0,0 +1,31 @@
+using NUnit.Framework;
+
+namespace Unity.Netcode.EditorTests
+{
+    public class XXHashTests
```

Comment:
We never had unit tests to verify hash results, I implemented them with some hardcoded strings and expected hardcoded results.

Having said that, even though it's a full reimplementation of the 3rd-party one, they both give us the exact same results as expected as they are both completely valid and accurate implementations — we're just getting rid of our external dependency and also making them just slightly more performant/efficient.